virttest/build_helper.py

```diff
@@ -688,6 +688,15 @@ def make_install(self):
 
     install = make_install
 
+    def make_rpm(self):
+        """
+        Run "make rpm"
+        """
+        os.chdir(self.build_dir)
+        process.system("make rpm")
```

Comment:
how do we ensure make rpm succeeded or got error out

Reply:
process.system will error out and should result in test failure